zone/zone.h

```diff
@@ -107,7 +107,7 @@ class Zone {
 	AA::Ability *GetAlternateAdvancementAbilityByRank(int rank_id);
 	AA::Rank *GetAlternateAdvancementRank(int rank_id);
 	bool is_zone_time_localized;
-	bool process_mobs_while_empty;
+	bool m_idle_when_empty;
```

Comment:
Keep in mind we use `m_` for `private` member variables. If we're going to prefix `m_` we should move it to private.

Then if we want to mutate it outside of the zone object, we use setters/getters